studies/templates/studies/study_responses.html

```diff
@@ -127,175 +88,101 @@
 <div class='container mb-lg'>
     <div class='row'>
         <div class="col-xs-12 col-sm-7 col-md-8 col-lg-9">
-            <a href="{% url 'exp:study-list' %}"> Manage Studies </a> &gt; <a href="{% url 'exp:study-detail' pk=study.id %}"> {{ study.name}} </a> &gt; Responses
+            <a href="{% url 'exp:study-list' %}"> Manage Studies </a> &gt; <a href="{% url 'exp:study-detail' pk=study.id %}"> {{ study.name}} </a> &gt; Individual Responses
```

Comment:
Really doesn't matter, but bootstrap does have a thing for [breadcrumbs](http://bootstrapdocs.com/v3.3.6/docs/components/#breadcrumbs).